pkg/apis/serving/v1alpha1/criticality.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2023 The KServe Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// This has been extracted as v1 does not utilise criticality, but v1alpha2 does.
```

Comment:
It was renamed: https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1363

You may be able to use the new type names, rather than extract the type.